Btms.Backend.IntegrationTests/Consumers/AmazonQueues/HmrcDecisionsConsumerTestsBase.cs

```diff
@@ -0,0 +1,41 @@
+using Btms.Types.Alvs;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Btms.Backend.IntegrationTests.Consumers.AmazonQueues;
+
+[Collection("AwsSqsSns")]
+[Trait("Category", "Integration")]
+public class HmrcDecisionsConsumerTestsBase : IAsyncLifetime
```

Comment:
What are HmrcDecisions?

The naming we've used to date is Cds for clearance requests and finalisations, ALVS for decisions.